frontend/mr-admin-flate/src/components/detaljside/Metadata.tsx

```diff
@@ -21,8 +22,10 @@ export function Metadata({
 }) {
   return (
     <div className={styles.header_og_verdi}>
-      <dt className={styles.bold}>{header}</dt>
-      <dd className={styles.definition}>{verdi}</dd>
```

Comment:
fikk klage fra a11y, fjernet derfor dt og dd osv

Reply:
Vi bør se på hvordan vi kan få til en mest mulig korrekt semantisk html-struktur for disse verdiene som er metadata.